src/clout/core.cljc

```diff
@@ -0,0 +1,162 @@
+(ns clout.core
+  "A small language for routing."
+  (:require [clojure.string :as string]
+            [clojure.set :as set]
+            [instaparse.core :as insta]
+            ))
+
+(def ^:private re-chars (set "\\.*+|?()[]{}$^"))
+
+#?(:clj
```

Comment:
Please avoid repetition, so in this case the reader conditional could be placed inside `re-escape` and we wouldn't need to copy the definition.